website/docs/library/index/edge.md

```diff
@@ -0,0 +1,74 @@
+---
+id: edge
+title: Edge mode
+sidebar_label: Edge mode
+slug: /library/index/edge
+---
+
+An H3 Edge index (mode 3) represents a single undirected edge between two cells. One of the two cells is arbitrarily picked as an "owner" or "origin" cell, which is used to calculate the canonical index of the edge. The components of the H3 Edge index are packed into a 64-bit integer in order, highest bit first, as follows:
```

Comment:
In the abstraction, no cell is the "owner" of the edge because it is just an unordered pair of adjacent cells. In the internal implementation, we choose an "owner", but I wouldn't say it is arbitrary. The "owner" is unambiguously the smaller of the two cells in the "canonical ordering".

Reply:
Thought: There should be no concept of "owner" or "origin" in the API for undirected edges, but it probably does make sense to talk about in terms of the implementation details. I'm not sure how best to allocate those levels of specificity in the docs, tho :/

Reply:
I removed the phrase "arbitrary" and explained the exact rule.

I think we need to explain how the canonical index is arrived at and why it represents the unordered edge. I would put further explanation of how to think about the mode and how to use it before we introduce the technical details of how it's encoded. Right now we don't have that content on this page. For the API, we don't commit to ordering for the most part, although I believe I do rely on that in some of the tests.